odoo/addons/base/models/res_users.py

```diff
@@ -1618,6 +1619,27 @@ def remove(self):
             return {'type': 'ir.actions.act_window_close'}
         raise AccessError(_("You can not remove API keys unless they're yours or you are a system user"))
 
+    def get_api_key(self, key):
+        index = key[:INDEX_SIZE]
+        self.env.cr.execute('SELECT id, key FROM res_users_apikeys WHERE index = %s', [index])
+        for [id, current_key] in self.env.cr.fetchall():
```

Comment:
That's just a style thing, but brackets are unnecessary. Also maybe `key_id` instead of `id`?